crates/api_common/src/community.rs

```diff
@@ -242,3 +243,45 @@ pub struct TransferCommunity {
 pub struct GetRandomCommunity {
   pub type_: Option<ListingType>,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct ListCommunityPendingFollows {
+  /// Only shows the unapproved applications
+  pub pending_only: Option<bool>,
+  // Only for admins, show pending follows for communities which you dont moderate
+  pub all_communities: Option<bool>,
+  pub page: Option<i64>,
+  pub limit: Option<i64>,
+}
```

Comment:
Should probably have a `skip_serializing_none`.

Reply:
Fixed